int16 targetBody
float64[] Pose
---
bool status


